lib/streamlit/commands/page_config.py

```diff
@@ -48,6 +54,9 @@ def set_page_config(
         How the sidebar should start out. Defaults to "auto",
         which hides the sidebar on mobile-sized devices, and shows it otherwise.
         "expanded" shows the sidebar initially; "collapsed" hides it.
+    menu_options: dict
+        Dictionary should be a string that maps to another string (URL). The
+        accepted strings are "Get Help", "Report a Bug", and "About".
```

Comment:
Nothing to do for now, but product will probably want to do some wordsmithing for all the public facing docstring changes down before this gets checked in